src/main/java/ua/ek/base/BasePage.java

```
@@ -22,36 +23,45 @@ public BasePage(WebDriver driver) {
         PageFactory.initElements(driver, this);
     }
 
-    public WebElement getEnterLink() {
+    protected WebElement getEnterLink() {
         if (isWebElementDisplayed(enterLink)) {
             return enterLink;
         }
         return null;
     }
 
-/*
     // Get text from web element
-    public static String getWebElementText(WebElement webElement){
-        return webElement.getText().trim();
+    protected String getWebElementText(By by){
+        if(isWebElementPresent(by)) {
+            return driver.findElement(by).getText().trim();
+        }else {
+            return "";
```

throw RuntimeException instead of returning of empty string

---

Or use waitUntilElementIsVisible method and return element text immediately